client/galaxy/scripts/components/Workflow/WorkflowList.vue

```diff
@@ -85,31 +85,27 @@ import { Services } from "./services.js";
 import WorkflowTags from "./WorkflowTags.vue";
 import WorkflowDropdown from "./WorkflowDropdown.vue";
 
+const NAME_FIELD = {key: "name", sortable: true};
+const TAGS_FIELD = {key: "tags", sortable: true};
+const BOOKMARK_FIELD = {key: "bookmark"};
+const EXECUTE_FIELD = {key: "execute", "label": ""};
+
+
 export default {
     components: {
         workflowtags: WorkflowTags,
         workflowdropdown: WorkflowDropdown
     },
+    props: {
+        simplified: {
+            type: Boolean,
+            default: false
+        }
+    },
     data() {
         return {
             error: null,
-            fields: [
-                {
-                    key: "name",
-                    sortable: true
-                },
-                {
-                    key: "tags",
-                    sortable: true
-                },
-                {
-                    key: "bookmark"
-                },
-                {
-                    key: "execute",
-                    label: ""
-                }
-            ],
+            fields: this.simplified ?  [NAME_FIELD, TAGS_FIELD, EXECUTE_FIELD] : [NAME_FIELD, TAGS_FIELD, BOOKMARK_FIELD, EXECUTE_FIELD],
```

Comment:
This might be easier to read and work with if we just create two dictionaries like `fieldsSimple` and `fieldsComplex` and then add a computed parameter name `fields` which returns one of them depending on the simplified flag.

Reply:
Good idea - sounds a lot cleaner! I will do this. Thanks.